settings/src/components/webauthn/webauthn.js

```diff
@@ -128,28 +113,85 @@ export default function WebAuthn() {
 
 			<p className="wporg-2fa__submit-actions">
 				<Button variant="primary" onClick={ () => setFlow( 'register' ) }>
-					Register New Key
+					Register new key
 				</Button>
 
 				{ keys.length > 0 && (
 					<Button
 						variant="secondary"
-						onClick={ webAuthnEnabled ? disableProvider : enableProvider }
+						onClick={ webAuthnEnabled ? showConfirmDisableModal : toggleProvider }
 						disabled={ statusWaiting }
 					>
-						{ webAuthnEnabled ? 'Disable Security Keys' : 'Enable Security Keys' }
+						{ `${ webAuthnEnabled ? 'Disable' : 'Enable' } security keys` }
 					</Button>
 				) }
-
-				{ statusWaiting && <Spinner /> }
 			</p>
 
+			{ statusWaiting && (
+				<p className="wporg-2fa__webauthn-register-key-status">
+					<Spinner />
```

Comment:
There's a bit of a weird loading state where the spinner shows up below the content and then the buttons leave. 

<img src="https://github.com/WordPress/wporg-two-factor/assets/1657336/4b953d89-7195-430a-8a0a-c895da277f4e" width="400" />

Reply:
I'll see if this can be smoothed out, but maybe in a follow up PR adding the processing animation from the designs.

Reply:
[I've simplified this](https://github.com/WordPress/wporg-two-factor/pull/223/commits/2634c00ce222f64195af980de117888e0b924ac6); there is now a message `Connecting...` with spinner, before the `Waiting...` message appears. This reduces the visual changes significantly